app/views/sessions/_card.html.erb

```diff
@@ -2,6 +2,17 @@
 
 <div class="flex flex-col w-full p-5 mb-4 bg-white rounded-[20px]">
   <div class="flex flex-col w-full">
+    <% if session.live? %>
+      <div class="h-8 px-4 py-2 bg-green-500 rounded-lg bg-opacity-[.08] justify-center items-center inline-flex mb-4">
+        <div class="grow shrink basis-0 h-4 justify-center items-center gap-2 flex">
+          <span class="relative flex h-4 w-4">
+            <span class="animate-slow-ping absolute inline-flex h-full w-full rounded-full bg-green-500 opacity-75"></span>
+            <span class="relative inline-flex rounded-full h-4 w-4 bg-green-500"></span>
+          </span>
+          <div class="grow shrink">Session is live now.</div>
```

Comment:
Couple of improvements:
1. No need to use flex grow and shrink. It should be enough with w-full for both the child and the flex parent, and use text-align: left.
2. This text seems a little bit bigger than in the design. Also, use an appropriate tag for text of this kind, such as p or span

Reply:
had to keep grow on parent
but child is now w-full